app/helpers/application_helper/button/host_feature_button_with_disable.rb

```diff
@@ -2,7 +2,6 @@ class ApplicationHelper::Button::HostFeatureButtonWithDisable < ApplicationHelpe
   needs :@record
 
   def visible?
-    return false if @record.kind_of?(ManageIQ::Providers::Openstack::InfraManager)
```

Comment:
Same here. The `HostFeatureButtonWithDisable` is used only on two places, so it may be removed completly:
```
app/helpers/application_helper/toolbar/host_center.rb
160:          :klass   => ApplicationHelper::Button::HostFeatureButtonWithDisable,
197:          :klass   => ApplicationHelper::Button::HostFeatureButtonWithDisable,
```
 I'm not sure if you can remove `HostFeatureButton`, it's used on more places:

```
app/helpers/application_helper/toolbar/host_center.rb
151:          :klass   => ApplicationHelper::Button::HostFeatureButton,
169:          :klass   => ApplicationHelper::Button::HostFeatureButton,
179:          :klass   => ApplicationHelper::Button::HostFeatureButton,
188:          :klass   => ApplicationHelper::Button::HostFeatureButton,
```